UdpHosts/GameServer/GameServer.csproj

```diff
@@ -35,6 +35,10 @@
         </None>
     </ItemGroup>
 
+    <ItemGroup>
+      <Folder Include="Packets\GSS\Generic\" />
+    </ItemGroup>
```

Comment:
Is this needed?

Reply:
Good find! No, removed it in #[75ba2c0](https://github.com/themeldingwars/PIN/pull/16/commits/75ba2c02057ae791f6781ece2764b24bc11c6b4b)